_includes/components/headline.html

```diff
@@ -1,4 +1,7 @@
 {% assign sdg_headline_data = site.data.headlines[page.indicator.slug] %}
+{% if site.data[page.language].headlines[page.indicator.slug] %}
+  {% assign sdg_headline_data = site.data[page.language].headlines[page.indicator.slug] %}
```

Comment:
I'm curious if you can assign `sdg_headline_data` before your `if` statement here.

Reply:
@SavvasStephanides I don't understand the question - maybe you could give an example?

Reply:
Sorry Brock, I had to have a closer look to see that the two `sdg_headline_data` declarations are different and not duplicate (line1 and line3).